apps/backup/netbackup/local/mode/dedupstatus.pm

```diff
@@ -63,9 +63,9 @@ sub set_counters {
                 closure_custom_threshold_check => \&catalog_status_threshold_ng
             }
         },
-        { label => 'usage', set => {
+        { label => 'usage', nlabel => 'disk_pool.deduplication.disk.usage.percentage', set => {
```

Comment:
need to remove the disk. i didn't see :)
disk_pool.deduplication.usage.percentage